distribution/test/itests/test-itests-ddf/src/test/java/ddf/test/itests/catalog/TestRegistry.java

```diff
@@ -342,5 +348,38 @@ private ValidatableResponse getCswRegistryResponse(String attr, String value) {
 
         return validatableResponse;
     }
+
+    public static String getCswRegistryInsert(String id, String regId) throws IOException {
+        return getCswInsert("rim:RegistryPackage", getRegistryNode(id, regId, regId));
+    }
+
+    public static String getCswRegistryUpdate(String id, String nodeName, String date, String uuid)
+            throws IOException {
+        return "<csw:Transaction\n" + "    service=\"CSW\"\n" + "    version=\"2.0.2\"\n"
+                + "    verboseResponse=\"true\"\n"
+                + "    xmlns:csw=\"http://www.opengis.net/cat/csw/2.0.2\">\n"
+                + "    <csw:Update typeName=\"rim:RegistryPackage\">\n" + getRegistryNode(id,
+                uuid,
+                uuid,
+                nodeName,
+                date) + "\n" + "    </csw:Update>\n" + "</csw:Transaction>";
+    }
+    public static String getRegistryNode(String id, String regId, String remoteRegId) throws IOException {
+        return getRegistryNode(id, regId, remoteRegId, "Node Name", "2016-01-26T17:16:34.996Z");
+    }
+
+    public static String getRegistryNode(String mcardId, String regId, String remoteReg, String nodeName, String date) throws IOException {
+        return AbstractIntegrationTest.getFileContent("/csw-rim-node.xml",
```

Comment:
This leading `/` as well. I bring these up cause Windows Compatibility page advises against them.